docs/api.rst

```diff
@@ -1307,7 +1307,7 @@ any consumers before it is terminated.
    :param string group_name: The name of the consumer group
    :param string instance: The ID of the consumer instance
 
-   :query timeout: The number of milliseconds for the underlying client library poll(timeout) request to fetch the records. Default to 5000ms.
+   :query timeout: The number of milliseconds for the underlying client library poll(timeout) request to fetch the records. Default is undefined. This parameter is used only if it's smaller than the `consumer.timeout.ms` that is defined either during consumer instance creation or in the proxy's config file.
```

Comment:
Sorry about noticing this late, but the original description is inaccurate as well. Do you mind if we change it while we're at it?

`The number of milliseconds for the underlying client library poll(timeout) request to fetch the records.` => `The maximum number of milliseconds that the proxy will spend fetching records`

Reply:
WDYT?